spec/support/acceptance/purge.rb

```diff
@@ -0,0 +1,4 @@
+def purge_katello
+  on default, 'yum -y remove foreman* tfm-*'
+  on default, "systemctl stop httpd", { :acceptable_exit_codes => [0, 5] }
+end
```

Comment:
For completeness, should this drop DBs in postgres as well?

Reply:
Fair question, I borrowed this originally from https://github.com/theforeman/puppet-foreman/blob/master/spec/support/acceptance/purge.rb which did not. To what degree we need to clean up is unclear (Redis? Certs?)